### PR TITLE
[UBS] - Bag, addTariffService() and editTariffService() refactoring

### PR DESCRIPTION
**Pull request by juseti**
dev
## JIRA

* [Main JIRA ticket](https://jira.softserve.academy/secure/RapidBoard.jspa?rapidView=id)

## Code reviewers

- [sergoliarnik](https://github.com/sergoliarnik)
- [JJJazl](https://github.com/JJJazl)
- [MaksymGolik](https://github.com/MaksymGolik)
- [Markiro1](https://github.com/Markiro1)

## Link to issue

https://github.com/ita-social-projects/GreenCity/issues/5120

## Summary of change

- In Bag.class deleted location field. Added necessary "nullable" marks.
- Added ch-drop-column-location-id-in-table-bag-Seti.xml changelog with location_id column removed.
- AddServiceDto.class renamed to TariffServiceDto.class. Added fields name, nameEng, description, descriptionEng; deleted fields TariffTranslationDto and locationId. TariffServiceDtoMapper.class refactored.
- EditServiceDto.class deleted.
- GetTariffServiceDto.class transfered from dto.tariff to dto.service in service-api package.
- Return type changed to GetTariffServiceDto for addTariffService() in SuperAdminService, SuperAdminServiceImpl and for createTariffService() in SuperAdminController.
- Deleted TariffTranslationDto.class
- Changed endpoint for createTariffService() to /{tariffId}/createTariffService in SuperAdminController.class.
- Fixed antMatchers endpoint /{tariffId}/createTariffService and added endpoints /includeBag/{id}, /excludeBag/{id} at configure(HttpSecurity http) in SecurityConfig.class.
- Refactored excludeBag() and includeBag() methods in SuperAdminServiceImpl.class
- Added ApiResponse(code = 404, message = HttpStatuses.NOT_FOUND) and ApiResponse(code = 400, message = HttpStatuses.BAD_REQUEST) for excludeBag() and includeBag() in SuperAdminController.class
- Added findBagsByLocationIdAndLocationStatusIsActive() method to BagRepository.class.
- Refactored getFirstPageData() in UbsClientServiceImpl.class
- Added BagTranslationDtoMapper.class.
- Deleted field locationId in BagTranslationDto.class
- Added message LOCATION_DOESNT_FOUND_BY_ID to ErrorMessage.class
- ModelUtils methods added or modified and unused methods deleted.
- Tests added to SuperAdminServiceImplTest, SuperAdminControllerTest, UBSClientServiceImplTest and to mapper-test classes.
- 

## Testing approach

Runing tests with and without params in swager by endpoints: 
/ubs/superAdmin + /{tariffId}/createTariffService, /{tariffId}/getTariffService, /deleteTariffService/{id}, /editTariffService/{id}, /excludeBag/{id}, /includeBag/{id}; /ubs/order-details.

### CODE REVIEW

File: core/src/main/java/greencity/controller/SuperAdminController.java [hunk 1]
```diff
@@ -62,33 +61,38 @@ class SuperAdminController {
...
-     * @param dto {@link AddServiceDto} dto for service.
-     * @return {@link AddServiceDto}
      * @author Vadym Makitra.
+     * @author Julia Seti
      */
-    @ApiOperation(value = "Create new tariff")
+    @ApiOperation(value = "Create new tariff service")
     @ApiResponses(value = {
-        @ApiResponse(code = 201, message = HttpStatuses.CREATED, response = AddServiceDto.class),
+        @ApiResponse(code = 201, message = HttpStatuses.CREATED, response = TariffServiceDto.class),
```
Comment (RomaMocherniuk) [inline core/src/main/java/greencity/controller/SuperAdminController.java]: I see you [return](https://github.com/ita-social-projects/GreenCityUBS/pull/990/files#diff-ea3792e71d2d7d18600958e51eae2e3f0316c18162b9d7e4ee58c871139b7058R83) `GetTariffServiceDto`. Why do you use `TariffServiceDto` here

File: core/src/main/java/greencity/controller/SuperAdminController.java [hunk 2]
```diff
@@ -145,9 +155,10 @@ public ResponseEntity<HttpStatus> deleteTariffService(
     @PreAuthorize("@preAuthorizer.hasAuthority('EDIT_PRICING_CARD', authentication)")
     @PutMapping("/editTariffService/{id}")
     public ResponseEntity<GetTariffServiceDto> editTariffService(
-        @RequestBody EditTariffServiceDto editTariff, @Valid @PathVariable Integer id,
+        @RequestBody TariffServiceDto dto,
+        @Valid @PathVariable Integer id,
```
Comment (RomaMocherniuk) [inline core/src/main/java/greencity/controller/SuperAdminController.java:159]: I see you use [Integer](https://github.com/ita-social-projects/GreenCityUBS/pull/990/files#diff-ea3792e71d2d7d18600958e51eae2e3f0316c18162b9d7e4ee58c871139b7058R159) and [long](https://github.com/ita-social-projects/GreenCityUBS/pull/990/files#diff-ea3792e71d2d7d18600958e51eae2e3f0316c18162b9d7e4ee58c871139b7058R110) for id. Why?

File: core/src/main/java/greencity/controller/SuperAdminController.java (hunk 1 above)
Comment (juseti) [reply core/src/main/java/greencity/controller/SuperAdminController.java]: fixed

File: core/src/main/java/greencity/controller/SuperAdminController.java (hunk 2 above)
Comment (juseti) [reply core/src/main/java/greencity/controller/SuperAdminController.java:159]: id has an integer type for Bag and a long type for TariffsInfo

**Review by sergoliarnik — APPROVED** [review]

**Review by MaksymGolik — APPROVED** [review]

**Review by JJJazl — APPROVED** [review]

**Review by Markiro1 — APPROVED** [review]

**Pull request merged**